common/wonders/wc_wonders_historical.txt

```diff
@@ -621,3 +620,105 @@ wonder_ulduar = {
 		}
 	}
 }
+
+#Janeiro Point
+wonder_janeiro_point = {
+	allow_construction = no
+	picture = GFX_wonder_strip_janeiro_point
```

Comment:
There are better pictures of the Booty Bay.

<details>
<summary>Click to expand</summary>

![](https://i.pinimg.com/originals/a4/eb/0a/a4eb0acdb10f96cdb0cffc76312d1079.jpg)
![](https://i.pinimg.com/originals/4e/cb/70/4ecb70911acda09366f09907577a5937.jpg)
![](https://external-preview.redd.it/3dy2-cHMJyssyVsvLU--WmeFZqfxPYVAKw-qbF8yxj8.jpg?auto=webp&s=6ad0b02865a062716141898e05979f7ca3dd88b4)

</details>